modules/node_modules/@ncigdc/dux/versionInfo.js

```diff
@@ -1,48 +1,33 @@
 // @flow
-/* global __GDC_API__, __COMMIT_HASH__, __VERSION_TAG__, __COMMIT_LINK__ */
-
 import { fetchApi } from '@ncigdc/utils/ajax';
 import { handleActions } from 'redux-actions';
-import packageJSON from './package.json';
+import packageJSON from '../../../../package.json';
```

Comment:
can stick with just the commit - I think the best solution is to check if a commit is a tag and then use the tag and if not show the commit. I just noticed that having a tag and a commit hash is confusing to people. but that can happen later